src/main/java/eu/europeana/metis/sandbox/consumer/workflow/CreatedConsumer.java

```diff
@@ -48,7 +48,7 @@ public void validateExternal(Event input) {
       var status = record.getErrors().isEmpty() ? Status.SUCCESS : Status.WARN;
       output = new Event(record, Step.VALIDATE_EXTERNAL, status);
     } catch (RecordProcessingException ex) {
-      LOGGER.error(ex.getMessage(), ex);
+      LOGGER.error("Exception while performing external validation step. ", ex);
```

Comment:
As Sonar remind me the message from the exception is already logged by the LOGGER